app/client/cli/keys.go

```diff
@@ -98,6 +99,12 @@ func NewKeysCommand() *cobra.Command {
 	cmd.AddCommand(signTxCmds...)
 	cmd.AddCommand(slipCmds...)
 
+	// Bind the store_child flag
+	if err := viper.BindPFlag("storeChild", cmd.Flags().Lookup("store_child")); err != nil {
```

Comment:
Seems inconsistent that we're using cobra flags for everything except for this one exception, which is only applicable to the `DeriveChild` function.

I think we should simply update the interface:

```go
// Before:
DeriveChildFromKey(parentAddr, pwd, index, childPwd, childHint)

// After: 
DeriveChildFromKey(parentAddr, pwd, index, childPwd, childHint, shouldStore)
```